framework/contracts/account/tests/token_transfers.rs

```diff
@@ -0,0 +1,306 @@
+    let interchain =
+        MockBech32InterchainEnv::new(vec![(SOURCE_CHAIN_ID, "source"), (DEST_CHAIN_ID, "dest")]);
+
+    let src = interchain.get_chain(SOURCE_CHAIN_ID)?;
+    let dst = interchain.get_chain(DEST_CHAIN_ID)?;
+
+    let src_abstr = Abstract::deploy_on_mock(src.clone())?;
+    let dest_abstr = Abstract::deploy_on_mock(dst.clone())?;
+
+    connect_one_way_to(
+        &src_abstr.call_as(&Abstract::mock_admin(&src)),
+        &dest_abstr.call_as(&Abstract::mock_admin(&dst)),
+        &interchain,
+    )?;
+
+    let ics20_channel = interchain
+        .create_channel(
+            SOURCE_CHAIN_ID,
+            DEST_CHAIN_ID,
+            &PortId::transfer(),
+            &PortId::transfer(),
+            "ics20-1",
+            Some(cosmwasm_std::IbcOrder::Unordered),
+        )?
+        .interchain_channel;
+
+    // Update the channels on the src side (for sending tokens)
+    src_abstr
+        .ans_host
+        .call_as(&Abstract::mock_admin(&src))
+        .update_channels(
+            vec![(
+                UncheckedChannelEntry {
+                    connected_chain: TruncatedChainId::from_chain_id(DEST_CHAIN_ID).to_string(),
+                    protocol: ICS20.to_string(),
+                },
+                ics20_channel
+                    .get_chain(SOURCE_CHAIN_ID)?
+                    .channel
+                    .unwrap()
+                    .to_string(),
+            )],
+            vec![],
+        )?;
+
+    // Update the channels on the src side (for sending back tokens)
+    dest_abstr
+        .ans_host
+        .call_as(&Abstract::mock_admin(&dst))
+        .update_channels(
+            vec![(
+                UncheckedChannelEntry {
+                    connected_chain: TruncatedChainId::from_chain_id(SOURCE_CHAIN_ID).to_string(),
+                    protocol: ICS20.to_string(),
+                },
+                ics20_channel
+                    .get_chain(DEST_CHAIN_ID)?
+                    .channel
+                    .unwrap()
+                    .to_string(),
+            )],
+            vec![],
+        )?;
+
+    let account = AccountI::new(ACCOUNT, src.clone());
+
+    account.instantiate(
+        &account::InstantiateMsg {
+            name: Some(String::from("first_account")),
+            description: Some(String::from("account_description")),
+            link: Some(String::from("https://account_link_of_at_least_11_char")),
+            namespace: None,
+            install_modules: vec![],
+            account_id: None,
+            owner: GovernanceDetails::Monarchy {
+                monarch: src.sender_addr().to_string(),
+            },
+            authenticator: None,
+        },
+        None,
+        &[],
+    )?;
+    account.set_ibc_status(true)?;
+    // Whitelist ibc_client to enable ibc-client callbacks
+    let ibc_client = account.module_address(IBC_CLIENT)?;
+    account.update_internal_config(account::InternalConfigAction::UpdateWhitelist {
```

Comment:
Instead of this operation, we need to whitelist the IBC client inside the account contract

Reply:
What do you mean by that?

Reply:
Ok, ibc-client whitelisted by default now!